includes/enrolment/class-sensei-course-manual-enrolment-provider.php

```diff
@@ -14,9 +14,10 @@
  *
  * @since 3.0.0
  */
-class Sensei_Course_Manual_Enrolment_Provider implements Sensei_Course_Enrolment_Provider_Interface {
-	const META_PREFIX_LEGACY_MIGRATION = 'course-enrolment-manual-legacy-';
-	const META_PREFIX_MANUAL_STATUS    = 'course-enrolment-manual-';
+class Sensei_Course_Manual_Enrolment_Provider
+	extends Sensei_Course_Enrolment_Stored_Status_Provider
+	implements Sensei_Course_Enrolment_Provider_Interface {
```

Comment:
It is a bit superfluous to implement it in this class and the parent class, but I'm keeping it here for now to be explicit. 

The parent class implements it because it uses `$this` on an argument that expects `Sensei_Course_Enrolment_Provider_Interface`, so we need to make sure `Sensei_Course_Enrolment_Stored_Status_Provider` always implements `Sensei_Course_Enrolment_Provider_Interface`